azure-intro/java/content/lab-03/App.java

```diff
@@ -0,0 +1,91 @@
+package myproject;
+
+import com.pulumi.Pulumi;
+import com.pulumi.azurenative.resources.ResourceGroup;
+import com.pulumi.azurenative.storage.StorageAccount;
+import com.pulumi.azurenative.storage.StorageAccountArgs;
+import com.pulumi.azurenative.storage.enums.Kind;
+import com.pulumi.azurenative.storage.enums.SkuName;
+import com.pulumi.azurenative.storage.inputs.SkuArgs;
+import com.pulumi.azurenative.storage.StorageAccountStaticWebsite;
+import com.pulumi.azurenative.storage.StorageAccountStaticWebsiteArgs;
+import com.pulumi.azurenative.storage.Blob;
+import com.pulumi.azurenative.storage.BlobArgs;
+import com.pulumi.asset.FileAsset;
+import com.pulumi.azurenative.web.AppServicePlan;
+import com.pulumi.azurenative.web.AppServicePlanArgs;
+import com.pulumi.azurenative.web.inputs.SkuDescriptionArgs;
+import java.util.Arrays;
+import com.pulumi.azurenative.web.WebApp;
+import com.pulumi.azurenative.web.WebAppArgs;
+import com.pulumi.azurenative.web.inputs.NameValuePairArgs;
+import com.pulumi.azurenative.web.inputs.SiteConfigArgs;
```

Comment:
With an IDE, or a production code situation, I whole-heartedly agree. Because this is a tutorial, and there's no guarantee that `import`s will be re-ordered automatically on save, I feel like it's more clear to keep them in the sequential order they're added throughout the tutorial.